juju/controller.py

```diff
@@ -529,40 +529,50 @@ async def get_models(self, all_=False, username=None):
         .. deprecated:: 0.7.0
            Use :meth:`.list_models` instead.
         """
-        controller_facade = client.ControllerFacade.from_connection(
-            self.connection())
-        for attempt in (1, 2, 3):
-            try:
-                return await controller_facade.AllModels()
-            except errors.JujuAPIError as e:
-                # retry concurrency error until resolved in Juju
-                # see: https://bugs.launchpad.net/juju/+bug/1721786
-                if 'has been removed' not in e.message or attempt == 3:
-                    raise
+        return await self.list_models(username, all_)
+
+    async def model_uuids(self, username=None, all_=False):
```

Comment:
```suggestion
    async def model_uuids(self, username=None, all_=False):
```

Why the underscore on `all_`?

Reply:
Because the `get_models` already had that parameter before this change so I just went with that. Why I kept it like that, is that I thought it's more like a flag rather than a value argument, so guess that made sense to me. Though I can change it if it doesn't make sense for others.